lib/commonOpts.ts

```diff
@@ -92,9 +92,9 @@ export function getCommandName(argv: Arguments<{ _?: string[] }>): string {
 export function getAccountId(
   options: Arguments<{ portal?: number | string; account?: number | string }>
```

Comment:
The reason this is failing for the install deps command is because options is a required field and it's not being passed in [here](https://github.com/HubSpot/hubspot-cli/blob/main/commands/project/installDeps.ts#L24). This will be easier to enforce once TS is rolled out everywhere, but we should keep the pattern of requiring options rather than allowing for it to not exist.